src/main/java/tc/oc/pgm/api/setting/SettingKey.java

```diff
@@ -1,21 +1,35 @@
 package tc.oc.pgm.api.setting;
 
+import static com.google.common.base.Preconditions.*;
 import static tc.oc.pgm.api.setting.SettingValue.*;
 
+import com.google.common.collect.ImmutableList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
 /**
  * A toggleable setting with various possible {@link SettingValue}s.
  *
  * @see SettingValue
  */
 public enum SettingKey {
   CHAT("chat", CHAT_TEAM, CHAT_GLOBAL, CHAT_ADMIN), // Changes the default chat channel
-  DEATH("death", DEATH_OWN, DEATH_ALL); // Changes which death messages are seen
+  DEATH(
+      Arrays.asList("death", "dms", "dm"),
```

Comment:
`/toggle dm` sounds like you are toggling private (direct) messages because when you get a private message it says `[DM]`.

Reply:
I have used old aliases from the 1.11 PGM.

https://github.com/OvercastNetwork/ProjectAres/blob/079d13ce62aee9a85f6871384a53356bd25735a8/PGM/src/main/java/tc/oc/pgm/death/DeathMessageSetting.java#L11

Reply:
Let's only support `death` and `dms`